CriticalMass/NextRidesRequest.swift

```diff
@@ -7,8 +7,8 @@ struct NextRideQuery: Codable {
     let centerLatitude: CLLocationDegrees
     let centerLongitude: CLLocationDegrees
     let radius: Int
-    let year = Date.getCurrent(\.year)
-    let month = Date.getCurrent(\.month)
+    var year = Date.getCurrent(\.year)
```

Comment:
Was this change intended?

Reply:
yes. there was a warning which I wanted to rid of.
Also I think this is the way to go when declaring properties that have a default value on structs